website_docs/getting-started.md

```diff
@@ -3,552 +3,897 @@ title: "Getting Started"
 weight: 2
 ---
 
-Welcome to the OpenTelemetry for Go getting started guide! This guide will walk you through the basic steps in installing, instrumenting with, configuring, and exporting data from OpenTelemetry. Before you get started, be sure to have Go 1.16 or newer installed.
+In this guide, you'll learn how to set up and get tracing telemetry from an HTTP
+server using Go.
 
-Understand how a system is functioning when it is failing or having issues is critical to resolving those issues. One strategy to understand this is with tracing. This guide shows how the OpenTelemetry Go project can be used to trace an example application. You will start with an application that computes Fibonacci numbers for users, and from there you will add instrumentation to produce tracing telemetry with OpenTelemetry Go.
+You can find more elaborate examples that use different OpenTelemetry libraries
+in [example subfolders for each
+library](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation)
 
-For reference, a complete example of the code you will build can be found [here](https://github.com/open-telemetry/opentelemetry-go/tree/main/example/fib).
+## Installation
 
-To start building the application, make a new directory named `fib` to house our Fibonacci project. Next, add the following to a new file named `fib.go` in that directory.
+To begin, you'll want to install OpenTelemetry and the `net/http`
+instrumentation package:
 
-```go
-package main
-
-// Fibonacci returns the n-th fibonacci number.
-func Fibonacci(n uint) (uint64, error) {
-	if n <= 1 {
-		return uint64(n), nil
-	}
-
-	var n2, n1 uint64 = 0, 1
-	for i := uint(2); i < n; i++ {
-		n2, n1 = n1, n1+n2
-	}
-
-	return n2 + n1, nil
-}
 ```
+go get go.opentelemetry.io/otel \
```

Comment:
This only works if we already have a mod file set up.  Would it make sense to have a `go mod init main` step before this?